AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -32,6 +33,29 @@ @interface ASRangeController ()
 
 @implementation ASRangeController
 
+#pragma mark - NSObject
+
++ (void)load {
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    [self registerLowMemoryNotification];
+  });
+}
+
+#pragma mark - Class
+
++ (ASWeakSet *)rangeControllers
+{
+  static ASWeakSet<ASRangeController *> *rangeController;
```

Comment:
should use plural.  Also, for a static variable, please prefix with __ (two underscores).  So probably __allRangeControllers.